trident-get-started/kubernetes-customize-deploy.adoc

```diff
@@ -78,9 +78,23 @@ metadata:
 spec:
   debug: true
   namespace: trident
-  tridentImage: netapp/trident:21.04.0
   imagePullSecrets:
   - thisisasecret
 ----
 
+Here is another exmaple that shows how Trident can be deployed with node selectors:
```

Comment:
@balaramesh I just noticed a typo. "exmaple" to "example". I will update this in an upcoming PR.